modules/optimization/doc/content/modules/optimization/examples/forceInv_pointLoads.md

```diff
@@ -0,0 +1,124 @@
+
+!listing test/tests/optimizationreporter/objective_gradient_minimize/point_loads/main.i
+         block=OptimizationReporter
+
+
+
+The optimization executioner on the main app does not need a mesh or variables and uses the action to set-up a dummy mesh and variables.
+
+!media media/fig_forward.png
+      style=width:60%;margin:auto;padding-top:2.5%;
+      id=figForward
+      caption=Optimization results for the (a) temperature field being matched at the $\times$ symbols in [figSetup] and (b) the intensity of the parameterized point loads applied at the $\bigcirc$ needed to produce the temperature field in (a)
+
+!media media/fig_adjoint.png
+      style=width:60%;margin:auto;padding-top:2.5%;
+      id=figAdjoint
+      caption=Adjoint temperature field and misfit load from first optimization iteration.
+
+The first step is to define the simulation to perform, in this case the simulation considered is a 2D
+steady state diffusion equation with Dirichlet boundary conditions on each end of the domain: find $u$
+such that
+
+!equation
+\frac{\partial u}{\partial t} + \frac{\partial^2 u}{\partial x^2} = 0 \quad x \in [0,1],
+
+where $u(0) = U(0, 0.5)$, $u(1) = U(1,2)$, and $U(a,b)$ defines a
+[continuous uniform distribution](https://en.wikipedia.org/wiki/Uniform_distribution_%28continuous%29)
+with $a$ and $b$ defining the lower and upper limits of the distribution, respectively.
+
+
+
+## Sub-Application Input
+
+The problem defined above, with respect to the [MultiApps](MultiApps/index.md) system, is a sub-application. The
+complete input file for the problem is provided in "main-app". The only item required
+to enable the stochastic analysis is the [Controls] block, which contains a
+[SamplerReceiver](/SamplerReceiver.md) object, the use of which will be explained
+in the following section.
+
+!listing test/tests/optimizationreporter/objective_gradient_minimize/point_loads/main.i
+         id=master_app
+         caption=Complete input file for executing the transient diffusion problem.
+
+!listing test/tests/optimizationreporter/objective_gradient_minimize/point_loads/forward.i
+        id=forward_app
+        caption=Complete input file for executing the transient diffusion problem.
+
+!listing test/tests/optimizationreporter/objective_gradient_minimize/point_loads/adjoint.i
+        id=adjoint_app
+        caption=Complete input file for executing the transient diffusion problem.
+
+## Master Input
+
+The master application, with respect to the [MultiApps](MultiApps/index.md) system, is the driver of the stochastic
+simulations, by itself it does not perform a solve. The complete input file for the master
+application is shown in [monte-carlo-master], but the import sections will be detailed individually.
+
+First, [Distributions] for each of the two stochastic boundary conditions are defined.
+
+!listing test/tests/optimizationreporter/objective_gradient_minimize/function_load/main.i block=OptimizationReporter
```

Comment:
This is using a different main file.

Reply:
Thanks @YaqiWang for reading this.  I'll fix these. I think the two md files you commented on are the only two that are worth looking at, the others are just place holders.

Reply:
This is really a good piece of work! I think having place holders for this initial setup of the module is fine.